### PR TITLE
Add missing quickActions

### PR DESCRIPTION
**Pull request by nnngrach**
https://github.com/osmandapp/OsmAnd-Issues/issues/361

### CODE REVIEW

File: Sources/Controllers/2.0/OAActionAddProfileViewController.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- (void) onProfileSelected:(NSArray *)items;
+
+@end
+
+@interface OAActionAddProfileViewController : OACompoundViewController
+
+@property (nonatomic) id<OAAddProfileDelegate> delegate;
+
+-(instancetype)initWithNames:(NSMutableArray<NSString *> *)names;
```
Comment (alex-osm) [inline Sources/Controllers/2.0/OAActionAddProfileViewController.h:23]: NSMutableArray > NSArray

File: Sources/Controllers/2.0/OAActionConfigurationViewController.mm [hunk 2]
```diff
@@ -414,6 +415,14 @@ - (void) addMapSource
     [self.navigationController pushViewController:mapSourceScreen animated:YES];
 }
 
+- (void) addProfile
+{
+    NSArray *arr = [_action getParams][@"stringKeys"] ? [NSMutableArray arrayWithArray:(NSArray *)[_action getParams][@"stringKeys"]] : [NSMutableArray new];
```
Comment (alex-osm) [inline Sources/Controllers/2.0/OAActionConfigurationViewController.mm:420]: [NSMutableArray new] > @[]

File: Sources/Plugins/SRTM/OASRTMPlugin.h [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+//  OASRTMPlugin.h
+//  OsmAnd
+//
+//  Created by nnngrach on 24.12.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OAPlugin.h"
+
+@interface OASRTMPlugin : OAPlugin
```
Comment (alex-osm) [inline Sources/Plugins/SRTM/OASRTMPlugin.h:11]: Looks like we do not need to create OASRTMPlugin to introduce quick action.

File: Sources/QuickAction/Actions/OAContourLinesAction.mm [hunk 4]
```diff
@@ -18,30 +18,31 @@ @implementation OAContourLinesAction
...
 {
-    return [parameter.value isEqual:@"disabled"] ? false : true;
+    return ![_parameter.value isEqual:@"disabled"];
 }
 
 - (void)execute
 {
-    parameter.value = ![self isContourLinesOn] ? [_settings.contourLinesZoom get] : @"disabled";
-    [styleSettings save:parameter];
+    _parameter = [_styleSettings getParameter:@"contourLines"];
```
Comment (alex-osm) [inline Sources/QuickAction/Actions/OAContourLinesAction.mm:43]: Why getParameter here? We have it on init.

File: Sources/QuickAction/Actions/OAShowHideMapillaryAction.m [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+- (NSString *)getActionStateName
+{
+    return [OsmAndApp instance].data.mapillary ? OALocalizedString(@"quick_action_mapillary_hide") : OALocalizedString(@"quick_action_mapillary_show");
+}
+
++ (OAQuickActionType *) TYPE
+{
+    if (!TYPE)
+        TYPE = [[OAQuickActionType alloc] initWithIdentifier:33 stringId:@"mapillary.showhid" class:self.class name:OALocalizedString(@"quick_action_showhide_mapillary_title") category:CONFIGURE_MAP iconName:@"ic_custom_mapillary_symbol" secondaryIconName:nil editable:NO];
```
Comment (alex-osm) [inline Sources/QuickAction/Actions/OAShowHideMapillaryAction.m:51]: mapillary.showhide

File: Sources/QuickAction/Actions/OASwitchProfileAction.m [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+- (void)execute
+{
+    OAAppSettings *settings = [OAAppSettings sharedManager];
+    NSArray<NSString *> *profiles = self.getParams[kStringKeys];
+    
+    if (profiles.count == 0)
+        return;
+    
+    BOOL showDialog = [[self getParams][KEY_DIALOG] boolValue]; //1?
```
Comment (alex-osm) [inline Sources/QuickAction/Actions/OASwitchProfileAction.m:49]: ?

File: Resources/en.lproj/Localizable.strings [hunk 7]
```diff
@@ -1201,6 +1202,19 @@
...
+"quick_action_showhide_mapillary_title" = "Show/hide Mapillary";
+"quick_action_showhide_mapillary_descr" = "A toggle to show or hide the Mapillary layer on the map.";
+"quick_action_mapillary_hide" = "Hide Mapillary";
+"quick_action_mapillary_show" = "Show Mapillary";
+"quick_action_remove_next_destination" = "Delete nearest destination point";
+"quick_action_remove_next_destination_descr" = "Deletes the next destination on your route. If that is the final destination, the navigation will stop.";
+"change_application_profile" = "Change app profile";
+"shared_string_add_profile" = "Add profile";
+"quick_action_switch_profile_descr" = "The action button switches between selected profiles.";
+"application_profiles" = "App profiles";
```
[the remark this answers is not in the document]
Comment (nnngrach) [reply Resources/en.lproj/Localizable.strings:1214]: done

File: Resources/en.lproj/Localizable.strings [hunk 8]
```diff
@@ -1201,6 +1202,18 @@
 "toggle_public_transport" = "Show/hide public transport";
 "public_transport_show" = "Show public transport";
 "public_transport_hide" = "Hide public transport";
+"quick_action_showhide_mapillary_title" = "Show/hide Mapillary";
+"quick_action_showhide_mapillary_descr" = "A toggle to show or hide the Mapillary layer on the map.";
```
Comment (PaulStets) [inline Resources/en.lproj/Localizable.strings:1206]: Tapping this action button shows or hides the Mapillary layer on the map. (always use phrases that are similar to other actions)

File: Resources/en.lproj/Localizable.strings [hunk 9]
```diff
@@ -1201,6 +1202,18 @@
...
+"quick_action_mapillary_hide" = "Hide Mapillary";
+"quick_action_mapillary_show" = "Show Mapillary";
+"quick_action_remove_next_destination" = "Delete nearest destination point";
+"quick_action_remove_next_destination_descr" = "Deletes the next destination on your route. If that is the final destination, the navigation will stop.";
+"change_application_profile" = "Change app profile";
```
Comment (PaulStets) [inline Resources/en.lproj/Localizable.strings:1211]: Change/select application profile. (do not use abbreviations in the ui)

File: Resources/en.lproj/Localizable.strings [hunk 10]
```diff
@@ -1201,6 +1202,18 @@
...
+"select_application_profile" = "Select app profile";
+"shared_string_add_profile" = "Add profile";
+"quick_action_switch_profile_descr" = "The action button switches between selected profiles.";
+"profile_type_user_string" = "User profile";
+"profile_type_osmand_string" = "OsmAnd profile";
```
Comment (PaulStets) [inline Resources/en.lproj/Localizable.strings:1216]: Duplicate string (see key osmand_profile). Delete either one

File: Sources/QuickAction/Actions/OASwitchProfileAction.m [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OASwitchProfileAction.h"
+#import "OAQuickActionType.h"
+#import "OAColors.h"
+#import "OAApplicationMode.h"
+#import "OAAppSettings.h"
+#import "OAQuickActionRegistry.h"
+//#import "OAQuickActionSelectionBottomSheetViewController.h"
```
Comment (PaulStets) [inline Sources/QuickAction/Actions/OASwitchProfileAction.m:15]: remove comment

File: Sources/Controllers/2.0/OAActionAddProfileViewController.m [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+    _data = [OAProfileDataUtils getDataObjects:[OAApplicationMode allPossibleValues]];
+}
+
+- (void)applyLocalization
+{
+    _titleView.text = OALocalizedString(@"select_application_profile");
+    [_doneButton setTitle:OALocalizedString(@"shared_string_done") forState:UIControlStateNormal];
+}
+
+-(UIView *) getTopView
```
Comment (PaulStets) [inline Sources/Controllers/2.0/OAActionAddProfileViewController.m:90]: If you are using the constraints, these methods should not be implemented: -(UIView *) getMiddleView, -(UIView *) getTopView

File: Sources/Profiles/OAProfileDataUtils.h [hunk 13]
```diff
@@ -0,0 +1,17 @@
+//
+//  OAProfileDataUtils.h
+//  OsmAnd
+//
+//  Created by nnngrach on 28.12.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
```
Comment (PaulStets) [inline Sources/Profiles/OAProfileDataUtils.h:7]: Add git revision

File: Resources/en.lproj/Localizable.strings [hunk 14]
```diff
@@ -1202,14 +1201,14 @@
 "toggle_public_transport" = "Show/hide public transport";
 "public_transport_show" = "Show public transport";
 "public_transport_hide" = "Hide public transport";
-"quick_action_showhide_mapillary_title" = "Show/hide Mapillary";
```
Comment (PaulStets) [inline Resources/en.lproj/Localizable.strings]: Revert this change and paste the correct text into quick_action_showhide_mapillary_descr
